lib/components/user/notification-prefs-pane.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { ButtonToolbar, ControlLabel, FormControl, FormGroup, ToggleButton, ToggleButtonGroup } from 'react-bootstrap'
+
+const allowedNotificationChannels = [
+  {
+    type: 'email',
+    text: 'Email'
+  },
+  {
+    type: 'sms',
+    text: 'SMS'
+  },
+  {
+    type: 'none',
+    text: 'Don\'t notify me'
+  }
+]
+
+/**
+ * User notification preferences pane.
+ */
```

Comment:
~Can you add a TODO here for handling the SMS verification stuff? We should do that in another PR.~ OK, I guess there is a TODO for that already.

Also, we need a TODO about field validation.